docs/scripting.md

```diff
@@ -592,7 +592,7 @@ In addition to a regular expression and callback, the `hear` and `respond` funct
 
 The most important and most common metadata key is `id`. Every Listener should be given a unique name (options.id; defaults to `null`). Names should be scoped by module (e.g. 'my-module.my-listener'). These names allow other scripts to directly address individual listeners and extend them with additional functionality like authorization and rate limiting.
```

Comment:
Reading through the documentation, I had some thoughts.
* I think it'd be nice to bold "every", as this tool will be much much more useful as external packages are updated.
* It seems like it would be a good idea to mention, that existing packages _will_ need to be updated to be able to take advantage of this. Maybe this particular document isn't the best location, but at least for a while, I would probably be helpful to point out that these features will require support from package authors.